GUI/Configuration.cs

```diff
@@ -101,6 +120,24 @@ private static Configuration LoadConfiguration(string path)
                 try
                 {
                     configuration = (Configuration) serializer.Deserialize(stream);
+                    // Set default list of columns if missing
+                    if (configuration.VisibleColumnNames == null || configuration.VisibleColumnNames.Count < 1)
+                    {
+                        configuration.VisibleColumnNames = new List<string>() {
+                            "Installed",
+                            "UpdateCol",
+                            "ReplaceCol",
```

Comment:
For those 3 columns, we don't let the user manipulate them.
`Installed` is never hidden, and `UpdateCol`as well as `ReplaceCol` are never addressed via `SetColumnVisible()`, nor shown/hidden based on the configuration, only based on "if replacements or updates exist", so I think we can let them out of this list?

Reply:
Removed.